content/docs/14-troubleshooting/30-pcg.md

```diff
@@ -121,4 +124,135 @@ While these are the most common errors, some other issues might be related to th
 
 
 4. If the problem persists, contact our support team by sending an email to [support@spectrocloud.com](mailto:support@spectrocloud.com).
-<br />
\ No newline at end of file
+<br />
+
+
+# Scenario - IP Address Not Assigned During PCG Creation
+
+When deploying a PCG in VMware vCenter, the IP address fails to get assigned to the new node. This can be caused by networking errors such as DHCP issues or improper execution of cloud-init.
+<br />
+
+## Debug Steps
+<br />
+
+1. Verify the network connectivity between the new node and the DHCP server. Ensure that the DHCP server is operational and has available IP addresses in its pool.
+
+
+2. Check the network configuration on the new node and ensure that it is set to obtain an IP address automatically via DHCP.
+
+
+3. Review the cloud-init logs on the new node for any errors or warnings related to the IP address assignment process.
```

Comment:
As I read each step and put my n00b hat on, I can't help but ask "How do I do this"? How does a user check the connectivity? Is it as simple as us suggesting a `ping` between the instances? How can they verify their network configuration is correct? What file and configuration is expected?

These instructions don't really help the reader, we can do better.